src/java-opentimelineio/src/test/java/io/opentimeline/TimeTransformTest.java

```diff
@@ -81,4 +83,13 @@ public void testComparison() {
         assertTrue(txForm.notEquals(txForm3));
         assertFalse(txForm.equals(txForm3));
     }
+
+    @Test
+    public void t() {
+        SerializableObject serializableObject = new SerializableObject();
+//        System.out.println(serializableObject.toJSONString(new ErrorStatus()));
+        SerializableObject serializableObject1 = new SerializableObject();
+        System.out.println(serializableObject.isEquivalentTo(serializableObject1));
+        System.out.println(serializableObject.isUnknownSchema());
+    }
```

Comment:
@meshula in case it helps

line 93 causes a crash. I think for some reason it isn't able to call the `is_unknown_schema` method in cpp.

Then if you comment out line 93 and uncomment line 90, it crashes again. Giving an error like this:

```
#
# A fatal error has been detected by the Java Runtime Environment:
#
#  SIGSEGV (0xb) at pc=0x00007fc848195b3f, pid=29443, tid=29466
#
# JRE version: OpenJDK Runtime Environment (11.0.7+10) (build 11.0.7+10-post-Ubuntu-2ubuntu218.04)
# Java VM: OpenJDK 64-Bit Server VM (11.0.7+10-post-Ubuntu-2ubuntu218.04, mixed mode, sharing, tiered, compressed oops, g1 gc, linux-amd64)
# Problematic frame:
# C  [libopentimelineio.so+0x52b3f]  opentimelineio::v1_0::SerializableObject::Writer::write(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, opentimelineio::v1_0::SerializableObject const*)+0x3f
#
# Core dump will be written. Default location: Core dumps may be processed with "/usr/share/apport/apport %p %s %c %d %P %E" (or dumping to /home/karthik/OpenTimelineIO/src/java-opentimelineio/core.29443)
#
# An error report file with more information is saved as:
# /home/karthik/OpenTimelineIO/src/java-opentimelineio/hs_err_pid29443.log
```

In case you'd like to reproduce this with Java, 

- [Install gradle](https://gradle.org/install/)
- clone the repo
- `cd` into the java-opentimelineio directory
- run `gradle test --tests io.opentimeline.TimeRangeTest`

Reply:
Here's the error log for the error I've shown above:
https://pastebin.com/Br9MUsK5